doc/user_guide/introduction.rst

```diff
@@ -0,0 +1,104 @@
+Introduction
+============
+
+Welcome to the Sales Channel Module of Tryton. Simply speaking, a sales
+channel is a way of intorduting products and services to the market
+so that they get the attention of potential buyers. A sales channel can be of
+different types; 1) Direct sales channel if a business sells directly to
+customers and 2) Indirect, if there are intermediary bodies in between,
+such as retailers and dealers who are involved in selling products to
+customers.
+
+In this module, we will learn about creating new sales channel and adding
+records to them.
+
+Before we can start, you will have to make Tryton Server running locally
```

Comment:
This documentation assumes that you have the module installed and working in your Tryton database.